firebase-config-interop/src/main/java/com/google/firebase/remoteconfig/interop/rollouts/RolloutAssignment.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.firebase.remoteconfig.interop.rollouts;
+
+import androidx.annotation.NonNull;
+import com.google.auto.value.AutoValue;
+import com.google.firebase.encoders.DataEncoder;
+import com.google.firebase.encoders.annotations.Encodable;
+import com.google.firebase.encoders.json.JsonDataEncoderBuilder;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+/** Model representing a single rollout assigned to an app instance at a point in time. */
+@Encodable
```

Comment:
@themiswang does this still need to be encodable? (fine to leave it for this release but maybe we add a note if it could be removed)

Reply:
Let's maybe just leave it there. In the future other sdk might want to leverage on that